src/lib/electricity-maps/index.ts

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable prettier/prettier */
+import * as dotenv from 'dotenv';
+import * as dayjs from 'dayjs';
```

Comment:
Not a huge fan of importing the whole package like this when there is ESM modules avaialble but I saw some other plugin used it as well so I guess it's fine.

Reply:
Yeah neither am I, I'm open to improvements here.

Reply:
In the app we use the ESM imports but it means you can't chain functions the same way. I think this is fine for now and if we have the time in the future we can improve it (or if someone else does, it's open source after all).

Reply:
Ah this package don't have ESM exports, I was thinking of date-fns. Sorry for the mixup.